### PR TITLE
[BEAM-5123] Testsuite do not run all the tests it should

### PR DESCRIPTION
**Pull request by VaclavPlajt**
Class import collision unearthed and fixed. New logs recording test suite test methods resolving added. Small cleanup (unused parameter, unwanted TODO).

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

File: sdks/java/extensions/euphoria/euphoria-core/src/test/java/org/apache/beam/sdk/extensions/euphoria/core/testkit/junit/TestSuiteRunner.java [hunk 1]
```diff
@@ -59,7 +59,9 @@ public TestSuiteRunner(Class<?> klass) throws Throwable {
     BeamRunnerWrapper runner =
         BeamRunnerWrapper.ofDirect().withAllowedLateness(Duration.ofHours(1));
     Class<?>[] testClasses = getAnnotatedClasses(klass);
+
     for (Class<?> testClass : testClasses) {
+      LOG.info(String.format("Testkit will schedule tests in '%s'.", testClass));
```
Comment (mareksimunek) [inline sdks/java/extensions/euphoria/euphoria-core/src/test/java/org/apache/beam/sdk/extensions/euphoria/core/testkit/junit/TestSuiteRunner.java]: Meanwhile exception messages doesnt provide good solution for formatting, logging methods do. I would prefer, String.format is uselesss here or is there some benefit?
`LOG.info("Testkit will schedule tests in {}.", testClass);`

File: sdks/java/extensions/euphoria/euphoria-core/src/test/java/org/apache/beam/sdk/extensions/euphoria/core/testkit/junit/TestSuiteRunner.java [hunk 2]
```diff
@@ -59,7 +59,9 @@ public TestSuiteRunner(Class<?> klass) throws Throwable {
     BeamRunnerWrapper runner =
```
Comment (mareksimunek) [inline sdks/java/extensions/euphoria/euphoria-core/src/test/java/org/apache/beam/sdk/extensions/euphoria/core/testkit/junit/TestSuiteRunner.java]: Its not used now, why is it there?

**Review by mareksimunek — CHANGES_REQUESTED** [review]

**Comment by mareksimunek** [comment]
Few minor changes and we are good to go :)

File: sdks/java/extensions/euphoria/euphoria-core/src/test/java/org/apache/beam/sdk/extensions/euphoria/core/testkit/junit/TestSuiteRunner.java (hunk 1 above)
Comment (VaclavPlajt) [reply sdks/java/extensions/euphoria/euphoria-core/src/test/java/org/apache/beam/sdk/extensions/euphoria/core/testkit/junit/TestSuiteRunner.java]: No benefit. Only the `info` method has unclear contract to me. I do not know how to log `Throwable` with custom message without putting `Throwable` as last formatting parameter. Something which can potentially break once SLF4J implementation changes.

File: sdks/java/extensions/euphoria/euphoria-core/src/test/java/org/apache/beam/sdk/extensions/euphoria/core/testkit/junit/TestSuiteRunner.java (hunk 2 above)
Comment (VaclavPlajt) [reply sdks/java/extensions/euphoria/euphoria-core/src/test/java/org/apache/beam/sdk/extensions/euphoria/core/testkit/junit/TestSuiteRunner.java]: Removed. Thanks.

**Pull request merged**